UnrealPlugin/ThirdPersonMPLocalDeploy.md

```diff
@@ -0,0 +1,108 @@
+* [Optional] {depot}/UnrealMarketplacePlugin  [This contains the PlayFab Unreal Marketplace plugin, which is not required for this guide, but almost certainly required for other PlayFab features]
+
+It is not required that you have all of these in the same location, but it is likely useful to do so, and for organization, this guide encourages you to do so.
+
+## Instructions
+
+### Local execution, no containers
+
+First, you will need to configure your LocalMultiplayerAgent to execute your server project. The first iteration will run the process directly on your local PC without any isolation.
+
+In Explorer, find and open the file: ```{depot}\MpsAgent\LocalMultiplayerAgent\MultiplayerSettings.json```. An abbreviated version of this file with the parts important to this guide are as follows [NOTE the escaped \\'s for paths in the json - this is a json file, and thus it's required to escape all \\'s as \\\\]:
+
+```json
+{
+  "RunContainer": false,
+  ...
+  "AssetDetails": [
+    {
+      "MountPath": "C:\\Assets",
+      "LocalFilePath": "<PATH TO ZIP>"
+    }
+  ],
+  ...
+  "ProcessStartParameters": {
+    "StartGameCommand": "<PATH TO EXE> -log"
+  },
+  "ContainerStartParameters": {
+    "StartGameCommand": "C:\\Assets\\ThirdPersonMPServer.exe -log",
+	...
+  }
+}
+```
+
+For the purposes of this guide, the parts of the json file obscured by ```...``` above, just utilize the project defaults. The purpose and values for the important fields are as follows:
+
+* RunContainer: For this guide, this will always be false. Setting this to true requires Docker.
+    * When true, the zip file in <PATH TO ZIP> must contain the entire project server release build
+	* When true, the ProcessStartParameters/StartGameCommand is ignored, and ContainerStartParameters/StartGameCommand is used instead
+	* When true, everything is built and run in a docker container, rather than on the local machine context
+	* This guide covers the scenario when RunContainer is false, so that we can more easily debug the server process
+	* Setting this to true requires a Shipping Server build, additional [Windows](https://docs.adamrehn.com/ue4-docker/configuration/configuring-windows-server) or [Linux](https://unrealcontainers.com/blog/identifying-application-runtime-dependencies/) DLLs, and [Docker for Windows](https://www.docker.com/products/docker-desktop)
+* AssetDetails/LocalFilePath: <PATH TO ZIP>
+	* This location must be fully defined, and a valid zip file must exist at this location
+	* When RunContainer==false, this file is ignored, but its existence is still required
+	* The zip file can be empty
+	* This requirement is a low priority polish item that hasn't been reconsidered yet
+	* For this example, this could be: ```{depot}\\empty.zip```
+	* For the author, this is: ```"M:\\depot\\GSDK\\ThirdPersonMPGSDK\\Binaries\\Win64\\ThirdPersonMPServer.zip```, with an empty zip file at that location
```

Comment:
"For the author" meaning?

Reply:
oh you mean where to find the file. Hm. Maybe just mention it this way - "the file referred to should exist in this location" or something?

Reply:
I already acknowledge that the file needs to exist at that location in earlier lines.
I'm trying to describe that was the value for me, on my machine (as the author), and that I used that value, and that there was an empty zip file at that location.

Reply:
I'm inclined to leave it in this case.

Reply:
yeah makes sense